packages/adapter-components/src/fetch/request/pagination/queue.ts

```diff
@@ -88,7 +88,7 @@ export class RequestQueue<ClientOptions extends string> {
       })
       nextArgs.forEach(arg => this.enqueue(arg))
     } catch (e) {
-      log.error('Error processing args (%s): %s, stack: %s', safeJsonStringify(args), safeJsonStringify(e), e.stack)
+      log.error('Error processing args (%s): stack: %s', safeJsonStringify(args), e.stack)
```

Comment:
we shouldn't remove it, we should revert to what it was in https://github.com/salto-io/salto/pull/6206
```suggestion
      log.error('Error processing args (%s): %s, stack: %s', safeJsonStringify(args), e, e.stack)
```